### ISSUE TITLE
NTR: Adenosine deaminase 2 activity low or absent (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8234]
New term request Adenosine deaminase 2 activity low or absent (lee231_180709140204): ADA2

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000562
- label: Reduced circulating adenosine deaminase-2 activity
- parents: [Abnormal enzyme concentration or activity (HP:0034684)](https://hpo.jax.org/app/browse/term/HP:0034684)
- pmid: PMID:24552285
- def: Concentration of adenosine deaminase 2 in the blood circulation below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8234
- synonyms: Reduced circulating ADA2 activity(NONE)
